### ISSUE TITLE
Form 21P-601: Accessibility issues - Description text not announced when user puts focus on input

### ISSUE THREAD

**Issue by nkutub** [department-of-veterans-affairs/va.gov-team#127761]
## BACKGROUND
Team BIO-HEART completed Staging Review for Form 21P-601 on 10/30/25. There were 38 total findings identified by VA Platform (8 were launch blocking, while 30 were non-launch blocking). 

## PROBLEM STATEMENT / GOAL

-: Check your eligibility 1, Waiver of Substitution, and anywhere else you have radio buttons preceded by a paragraph
- Device: All, most prominently desktop screen readers (where users are most likely to use a keyboard to navigate)
- Orientation: Both portrait and landscape
- 
Some radio button groups aren’t answerable without reading the description above. For example, on “Waiver of substitution,” users can’t answer “Do you want to waive your right to substitution?” without knowing what those rights are, which are contained in the paragraphs above.
Screen reader users often use TAB to find the interactive fields in a form and quickly fill them out. This means that they would miss the description completely, and not have the necessary context to answer the question.

**Issue Severity**
- Expected WCAG failure: n/a (this isn’t a WCAG issue)
- Expected 508 audit severity: n/a
- Expected CC severity level: a11y-defect-2 (Labels or calls to action must be descriptive to assistive devices)
- Expected experience standard: User can complete necessary flows

**How to recreate:**
- Turn on a desktop screen reader like JAWS or VoiceOver
- On “Waiver of substitution,” TAB until you get to the “Yes” input - note that you skip the description entirely
[Clip](https://coformaco.zoom.us/clips/share/FuwZhFtXTzm4mqnsXhawqw) (using JAWS)

**Recommendation:** 
First, evaluate pages within radio buttons + descriptions and determine if a user needs that description to answer the question.
On those pages where they do, I recommend using the VADS radio button component with description message, and put the description text (“If you’ve filled out…”) in the message-aria-describedby slot: https://design.va.gov/storybook/?path=/story/uswds-va-radio--with-description-message
